tempto-core/src/main/java/io/prestosql/tempto/internal/listeners/ProgressLoggingListener.java

```diff
@@ -91,12 +91,7 @@ public void onTestSkipped(ITestResult testCase)
     private void logTestEnd(ITestResult testCase, String outcome)
     {
         long executionTime = currentTimeMillis() - testStartTime;
-        if (executionTime < 1000) {
-            LOGGER.info(outcome);
```

Comment:
It was logging test name before, but it wasn't logging duration. Is this what you need? If so please update commit message? And why you care about such low durations?

Reply:
Or what do I miss here?

Reply:
oh.. I see.

Maybe let's update this to:

```
            LOGGER.info("{}     /    {}", outcome, formatTestName(testCase));
```
?

Reply:
Logging duration is a consciously allowed, not intended side-effect of my change.
If you see a reason _not to_ log the duration when it's under 1s, i can restore the previous behavior.

Reply:
I don't really care much about duration. It just pollutes logging.